## Module description

The `nrps_pks_domains` module annotates domains with poly keytide synthase (PKS)
and non-ribosomal peptide synthase (NRPS) related functions.

## Annotated domains
| Domain name | Icon | Description | Source |
| ----------- | :--: | ----------- | ------ |
|ACP | | Acyl-carrier protein domain| [Medema et al. 2011](#Medema2011)
|ACP_beta | | Beta-branching acyl-carrier protein domain|SMART
|ACPS | | 4'-phosphopantetheinyl transferase| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF01648)
|Aminotran_1_2 | | Aminotransferase class I and II| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00155)
|Aminotran_3 | | Aminotransferase class III| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00202)
|Aminotran_4 | | Aminotransferase class IV| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF01063)
|Aminotran_5 | | Aminotransferase class V| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00266)
|AMP-binding | A | Adenylation domain| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00501)
|A-OX | A | Adenylation domain with integrated oxidase| [Medema et al. 2011](#Medema2011)
|B | B | Branching domain|[Blin et al. 2017](#Blin2017)
|Beta_elim_lyase | SH | Beta-eliminating lyase |[Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF01212)
|CAL_domain | CAL | Co-enzyme A ligase domain| [Medema et al. 2011](#Medema2011)
|cAT| cAT | Choline/Carnitine o-acyltransferase |[Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00755)
|Cglyc | C | Glycopeptide condensation domain| [Medema et al. 2011](#Medema2011)
|cMT | cMT | Carbon methyltransferase|[Ansari et al. 2008](#Ansari2008)
|Condensation | C | Condensation domain| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00668)
|Condensation_DCL | C | Condensation domain linking a L-amino acid to a peptide ending with a D-amino acid| [Rausch et al. 2007](#Rausch2007)
|Condensation_Dual | C | Dual condensation/epimerisation domain| [Rausch et al. 2007](#Rausch2007)
|Condensation_LCL | C | Condensation domain linking a L-amino acid to a peptide ending with a L-amino acid| [Rausch et al. 2007](#Rausch2007)
|Condensation_Starter |C | Starter condensation domain| [Rausch et al. 2007](#Rausch2007)
|ECH | ECH | Enoyl-CoA hydratase/isomerase|[Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00378)
|Ene_KS | KS | Enediyine ketosynthase|[Yadav et al. 2009](#Yadav2009)
|Epimerization domain |E | Epimerization domain| [Weber et al. 2009](#Weber2009)
|F | F | Formylation domain|[Blin et al. 2017](#Blin2017)
|FkbH | FkbH | FkbH-like domain|[Blin et al. 2017](#Blin2017)
|GNAT | GNAT | Gcn5-related N-acetyltransferase | [Blin et al. 2017](#Blin2017)
|Hal | Hal | Halogenase domain|[Blin et al. 2017](#Blin2017)
|Heterocyclization | C | Heterocyclization domain| [Medema et al. 2011](#Medema2011)
|Hyb_KS | KS | Hybrid ketosynthase|[Yadav et al. 2009](#Yadav2009)
|Itr_KS | KS | Iterative ketosynthase|[Yadav et al. 2009](#Yadav2009)
|LPG_synthase_C | | Phosphatidylglycerol lysyltransferase, C-terminal|[Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF09924)
|Mod_KS | KS | Modular ketosynthase|[Yadav et al. 2009](#Yadav2009)
|NAD_binding_4|NAD|Male sterility protein|[Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF07993)
|nMT | nMT | Nitrogen methyltransferase|[Ansari et al. 2008](#Ansari2008)
|NRPS-COM_Cterm | | NRPS COM domain C-terminal| [Medema et al. 2011](#Medema2011)
|NRPS-COM_Nterm | | NRPS COM domain N-terminal| [Medema et al. 2011](#Medema2011)
|oMT | oMT | Oxygen methyltransferase|[Ansari et al. 2008](#Ansari2008)
|PCP | | Peptidyl-carrier protein domain| [Medema et al. 2011](#Medema2011)
|PKS_AT | AT | Acyltransferase domain|[SMART](http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00827)
|PKS_DH | DH | Dehydratase domain|[SMART](http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00826)
|PKS_DH2 | DH | Dehydratase domain (avoids false positive PS domains)|
|PKS_DHt | DHt | Dehydratase domain variant more commonly found in trans-AT PKS clusters |
|PKS_Docking_Cterm | | PKS C-terminal docking domain| [Medema et al. 2011](#Medema2011)
|PKS_Docking_Nterm | | PKS N-terminal docking domain| [Medema et al. 2011](#Medema2011)
|PKS_ER | ER | Enoylreductase domain|[SMART](http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00829)
|PKS_KR | KR | Ketoreductase domain|[SMART](http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00822)
|PKS_KS | KS | Ketosynthase domain|[SMART](http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00825)
|PKS_PP | | Phosphopantetheine acyl carrier protein group (new in 5.1.1)| [SMART](http://smart.embl.de/smart/do_annotation.pl?DOMAIN=SM00823)
|Polyketide_cyc2 | | Polyketide cyclase/dehydratase| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF10604)
|Polyketide_cyc | | Polyketide cyclase| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF03364)
|PS | PS | Pyran synthase domain|[Blin et al. 2017](#Blin2017)
|PT | PT | Product template domain|[TIGRFams](https://www.ebi.ac.uk/interpro/entry/tigrfams/TIGR04532/)
|SAT | SAT | Starter unit:ACP transacylase in aflatoxin biosynthesis| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF16073)
|TD | TD | Terminal reductase domain| [Medema et al. 2011](#Medema2011)
|Thioesterase | TE | Thioesterase domain| [Pfam](https://www.ebi.ac.uk/interpro/entry/pfam/PF00975)
|TIGR01720 | |NRPS domain of unknown function|[TIGRFams](https://www.ebi.ac.uk/interpro/entry/tigrfams/TIGR01720/)
|TIGR02353||NRPS terminal domain of unknown function|[TIGRFams](https://www.ebi.ac.uk/interpro/entry/tigrfams/TIGR02353/)
|Tra_KS | KS | Trans-acyltransferase ketosynthase|[Yadav et al. 2009](#Yadav2009)
|Trans-AT_docking | | Trans-acyltransferase docking domain| [Medema et al. 2011](#Medema2011)
|X | X | Glycopeptide-specific NRPS domain responsible for oxygenase recruitment| [Medema et al. 2011](#Medema2011)

## Trans-AT domain subtypes
Ketosynthase domains (KS) of trans-acyltransferase polyketide synthases (Trans-AT-PKS)
have additional to their substrate specificities, i.e., the monomer incorporated into the module before the KS.

Canonical double bonds are located between the alpha and beta position.

Two modes of shifted double bond formation have been described, including a dedicated module for double bond shift and the presence of a characteristic DH-like domain.
KS domains with substrate specificities for shifted double bonds usually group with KS domains that show substrate specificity for completely reduced moieties.

Domains predicted to be non-elongating will have a prefix of `non-elongating-` within antiSMASH.
Ambiguous results between categories will be separated by a `/`, e.g. `beta-OH/keto`.

All the following categories are predicted by profiles from TransATor ([Helfrich et. al. 2019](#Helfrich2019)).


| Name | Specificity |
| ---- | ----------- |
|AA | Glycine introduced by the NRPS module upstream.|
|AcST | Acetyl groups as the starting building block of the polyketide. |
|alpha-D-Me-shDB | Alpha-methyl groups with beta-gamma-double bonds.|
|alphabeta-OH | Alpha-hydroxyl groups in conjunction with beta-hydroxyl groups.|
|alphaMe | Alpha-methyl groups with either a reduced bond or a beta-gamma-double bond.|
|alphaMe-beta-D-OH | Alpha-L-methyl groups in conjunction with beta-D-hydroxyl groups.|
|alphaMe-beta-L-OH | Alpha-methyl groups in conjunction with beta-L-hydroxyl groups.|
|alphaMe-betaOH | Alpha-methyl groups in conjunction with beta-hydroxyl groups.|
|alphaMe-DB | Alpha-methyl groups with double bonds.|
|alphaMe-eDB | Alpha-methyl groups with E-configured double bonds.|
|alphaMe-zDB | Alpha-methyl groups with Z-configured double bonds|
|arST | Aromatic rings as the starting building block of the polyketide. |
|beta-D-OH | Beta-D-hydroxyl groups.|
|beta-D-OMe | Beta-methoxy groups.|
|beta-Me | Either beta-exomethylene groups or reduced beta-methyl groups, depending on the module composition upstream.|
|beta-L-OH | Beta-L-hydroxyl groups.|
|beta-OH | Beta-hydroxyl groups.|
|br | This type is specific for vinylogous chain branching.|
|DB | Double bonds of various configurations.|
|eDB | E-configured double bonds.|
|exomethylene | Exomethylene groups. |
|keto | Beta-keto groups.|
|lacST | Lactate as the starting building block of the polyketide. |
|MeOST | Methoxycarbonyl units as the starting building block of the polyketide. |
|Miscellaneous | This type is elongating, but the substrate specificity can not be predicted.|
|non-elongating | When without a suffix, this type is non-elongating, but the substrate specificity can not be predicted.|
|OUT | This type is in an out group.|
|Oxa | Amino acids containing oxazole or thiazole rings introduced by the NRPS module upstream.|
|OxI | Substrates with inserted oxygen, oftentimes resulting in oxidative cleaving. |
|pyr | Pyran or furan rings, depending on the presence of an in-trans-acting hydroxylases two modules upstream. |
|red | Reduced bonds. |
|shDB | Beta-gamma-double bonds. |
|ST | Amidated amino acid starters. Amide groups are introduced by a dedicated aminotransferase. |
|unST | Phosphoglycerate-derived molecules as the starting building block of the polyketide. |
|zDB | Z-configured double bonds.|

<span id="Rausch2007">
1. Rausch, C., Hoof, I., Weber, T., Wohlleben, W. and Huson, D.H. (2007)
Phylogenetic analysis of condensation domains in NRPS sheds light on their functional evolution.
<emph>BMC Evol. Biol.</emph>, vol. <strong>7</strong>, 78.
</span>

<span id="Medema2011">
2. Medema, M.H., Blin, K., Cimermancic, P., de Jager, V., Zakrzewski, P., Fischbach, M.A., Weber, T., Takano, E., and Breitling, R. (2011)
antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences.
<emph>Nucl. Acids Research</emph>, vol. <strong>39</strong>, W339-W346.
</span>

<span id="Weber2009">
3. Weber, T., Rausch, C., Lopez, P., Hoof, I., Gaykova, V., Huson, DH., Wohlleben, W. (2009)
CLUSEAN: a computer-based framework for the automated analysis of bacterial secondary metabolite biosynthetic gene clusters.
<emph>J. Biotechnol.</emph>, vol. <strong>140</strong> 13-17.
</span>

<span id="Yadav2009">
4. Yadav, G., Gokhale, R.S. and Mohanty, D. (2009)
Towards prediction of metabolic products of polyketide synthases: an in silico analysis.
<emph>PLoS Comput. Biol.</emph>, vol. <strong>5</strong>, e1000351.
</span>

<span id="Blin2017">
5. Blin,K., Wolf,T., Chevrette,M.G., Lu,X., Schwalen,C.J., Kautsar,S.A., Suarez Duran,H.G., de Los Santos,E.L.C., Kim,H.U., Nave,M., et al. (2017)
antiSMASH 4.0-improvements in chemistry prediction and gene cluster boundary identification.
<emph>Nucleic Acids Res.</emph>, vol. <strong>45</strong>, W36–W41.
</span>

<span id="Ansari2008">
6. Ansari, M.Z., Sharma, J., Gokhale, R.S. and Mohanty, D. (2008)
In silico analysis of methyltransferase domains involved in biosynthesis of secondary metabolites.
</emph>BMC Bioinformatics</emph>, vol. <strong>9</strong>, 454
</span>

<span id="Helfrich2019">
7.   Helfrich, E.J.N., Ueoka, R., Dolev, A., Rust, M., Meoded, R.A., Bhushan, A., Califano, G., Costa, R., Gugger, M., Steinbeck, C., Moreno, P., Piel, J. (2019)
Automated structure prediction of trans-acyltransferase polyketide synthase products.
<emph>Nat. Chem. Biol.</emph>, vol. <strong>15</strong>, 813–821.
</span>
